npm/iptm/iptm_test.go

```diff
@@ -7,77 +7,145 @@ import (
 	"github.com/Azure/azure-container-networking/npm/metrics"
 	"github.com/Azure/azure-container-networking/npm/metrics/promutil"
 	"github.com/Azure/azure-container-networking/npm/util"
+	testutils "github.com/Azure/azure-container-networking/test/utils"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	initCalls = []testutils.TestCmd{
```

Comment:
I think it is really good to divide this initCalls slide into three slices according to "AddAllChains()", "CheckAndAddForwardChain()", and "AddAllRulesToChains()". Then merging them into initCalls. It will help better readability and maintenance and help reviewers to understand codes.